.github/workflows/test.yml

```diff
@@ -47,20 +57,27 @@ jobs:
           python-version: ${{ matrix.python }}
       - name: Install
         run: |
-          python -m pip install --upgrade pip setuptools wheel tox
+          python -m pip install --upgrade pip setuptools wheel tox tox-gh-actions
       - name: Test
-        run: tox
+        run: tox -q -- -q --cov-report=
```

Comment:
the `--cov-report=` looks like it's missing something. I know it's not, but might be worth a comment to point out that it suppresses terminal output

Reply:
See https://pytest-cov.readthedocs.io/en/latest/reporting.html

`--cov-report=` skips printing to the terminal because doing so removes the coverage file, which we need for the GitHub Action summary